modules/ossm-config-sampling.adoc

```diff
@@ -23,7 +23,7 @@ The Envoy proxy sampling rate applies for applications that are available to a S
 
 The Jaeger remote sampling rate applies to applications that are external to the Service Mesh, and do not use the Envoy proxy, such as a database. This sampling rate determines how much data the distributed tracing system collects and stores.
 ifdef::openshift-enterprise[]
-For more information, see xref:../../observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-configuring.adoc#distr-tracing-config-sampling_deploying-distributed-tracing-platform[Distributed tracing configuration options].
+For more information, see link:https://docs.openshift.com/container-platform/latest/observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-configuring.html#distr-tracing-config-sampling_distr-tracing-jaeger-configuring[Distributed tracing sampling configuration options].
```

Comment:
We need to swap from links that link to our internal docs to xrefs. The reason for this is that, if customers are on the portal and they click this docs.openshift.com link, they'll be redirected to docs.opc.com. this is a bad UX. These xrefs will need to be put into the associated assembly, since xrefs cannot go into modules.

Reply:
@smunje1 - It's incorrect here to use either an xref or a hard link. Both are incorrect. We can't use xrefs in modules, and we can't hard link to docs.openshift or access.redhat to circumvent this restriction. So, this needs to be changed back to an xref with a correct path even though using an xref is also incorrect.